### DISCUSSION TITLE
Why the svchosts grouping was disabled?

### DISCUSSION THREAD

**Discussion by EekaDroid** [farag2/Sophia-Script-for-Windows#156] [Q&A] [answered]
Hello farag2,

First of all, thanks for your continuous effort in developing this feature for us. My system is stable and way more usable ever since i started using it.

I could notice though, that running the script again (v5.9) in a new VM did not lead me to the processes number reduction as it did in my main PC, in which i have about 65 to 70 processes right after the startup.

Looking into the old script i used back then (v4.5.7) i noticed a feature was [removed](https://github.com/farag2/Windows-10-Sophia-Script/releases/tag/4.6), "Group svchost.exe processes". It is responsible for a great deal of background processes removal.

I could simply add it back into the latest script in order to have the feature, but i wanted to ask you why it was removed? Does it cause any risk for the system performance/stability due to the reduction of the amount of svchost processes available? Is it recommended to revert this feature in my main PC?

**Comment by farag2** [comment, marked as answer]
Hello!

>Does it cause any risk for the system performance/stability due to the reduction of the amount of svchost processes available? Is it recommended to revert this feature in my main PC?

I really dont remember why I've removed it. But if I had removed and didn't re-added it there is a reason for it. I was told to remove it by our forum member. Anyway by reducing CPU processes you dont make OS work better. It only adds instability, because if a process hangs it will bring with it all subprocesses. Thats why its better to have more similar processes than less. I can see here and there on YouTube that many YTers tell their auditory that e.g. they reduced Windows 10 processes from N to N-30. LOL. There is a great delusion to consider the number of processes as a performance criterion. But who cares? Every YT considers it his duty to create his own so called debloat script and PR it on YT pouring banana oil in his auditory's ear that only now their PC will speed up. LOL. Thats not true. So just relax, use your PC and dont chase a rainbow. 🌈 🏳️‍🌈 
Windows 10 works as it was designed. You cannot speed it up. Just to slow it down or better fine-tune and customize. 👍

**Reply by EekaDroid** [reply]
Thank you for your answer @farag2

Yea, i thought it was the cause, after researching a little about it.

I also noticed that there was no difference in ram usage from a clean Win10 install with this feature enabled or disabled (about 60 vs 90-110 opened processes).

Thanks again for this great tool and for sharing a little more knowledge. Stay safe!
